packages/blockchain-wallet-v4-frontend/src/components/Form/CountdownTimer/template.js

```diff
@@ -19,13 +19,27 @@ const Wrapper = styled.div`
 
 const CountdownTimer = props => (
   <Wrapper {...props}>
-    <Text size='11px' weight={400} color='gray-3'>
-      <FormattedMessage
-        id='scenes.exchange.secondstep.expiry_text'
-        defaultMessage='This quote will refresh in: {timeLeft}'
-        values={{ timeLeft: props.timeLeft }}
-      />
-    </Text>
+    {!props.payProInvoice ? (
```

Comment:
Instead of this (and instead of hardcoding the text 'This quote...'). It would be good to require this component take some type of property like countdownMessage (?) and pass that in from the parent component that uses CountdownTimer:

PARENT:
```
const countdownMessage = <Text
        size='14px'
        weight={500}
        color={getCountdownColor(props.expiryDate)}
      >
        <FormattedMessage
          id='modals.sendbtc.firststep.paypro_expiry_text'
          defaultMessage='Remaining time {timeLeft}'
          values={{ timeLeft: props.timeLeft }}
        />
      </Text>

...

<CountdownTimer countdownMessage={countdownMessage} ... ./>
```

Reply:
Reason it was done like this might be the way this formatted message element works, I'm getting the error that 
Messages must be statically evaluate-able for extraction.
  36 |         <FormattedMessage
  37 |           id='modals.sendbtc.firststep.paypro_expiry_text'
> 38 |           defaultMessage={countdownMessage + timeLeft}

Not sure how to do this honestly, can take another stab at it later

Reply:
Yes you're right about FormattedMessage but countdownMessage would be a react component (`Text`).

Reply:
Looking at doing this now, but I've also got the countdown timer on the second step of the send flow though so I'd have to create two new countdownMessage react components in each respective step and find a way to handle the getCountdownColor without including that function in both steps as well. Since both steps use the same countdown message feel like its better to keep it as a singular thing in the countdowntimer template.